### PR TITLE
create index in Search Mode tests added

### PR DESCRIPTION
**Pull request by AlenaNaboka**

### CODE REVIEW

File: tests/e2e/pageObjects/browser-page.ts [hunk 1]
```diff
@@ -127,6 +128,9 @@ export class BrowserPage {
     formatSwitcher = Selector('[data-testid=select-format-key-value]', { timeout: 2000 });
     formatSwitcherIcon = Selector('img[data-testid^=key-value-formatter-option-selected]');
     selectIndexDdn = Selector('[data-testid=select-index-placeholder],[data-testid=select-search-mode]', { timeout: 1000 });
+    createIndex = Selector('[data-testid=create-index-btn]');
```
Comment (vlad-dargel) [inline tests/e2e/pageObjects/browser-page.ts:131]: I suggest to rename it to 'createIndexBtn'

Comment (vlad-dargel) [reply tests/e2e/pageObjects/browser-page.ts:131]: and the same is for buttons below

File: tests/e2e/tests/critical-path/browser/search-capabilities.e2e.ts [hunk 2]
```diff
@@ -137,3 +137,50 @@ test
...
+        // Verify that user can create an index with multiple prefixes
+        await t.click(browserPage.indexNameInput);
+        await t.typeText(browserPage.indexNameInput, indexName);
+        await t.click(browserPage.prefixFieldInput);
+        await t.typeText(browserPage.prefixFieldInput, 'device:');
+        await t.pressKey('enter');
+        await t.typeText(browserPage.prefixFieldInput, 'mobile_');
+        await t.pressKey('enter');
+        await t.typeText(browserPage.prefixFieldInput, 'user_');
+        await t.pressKey('enter');
```
Comment (vlad-dargel) [inline tests/e2e/tests/critical-path/browser/search-capabilities.e2e.ts:177]: I can't see any 'expect' here

File: tests/e2e/pageObjects/browser-page.ts (hunk 1 above)
Comment (AlenaNaboka) [reply tests/e2e/pageObjects/browser-page.ts]: fixed

File: tests/e2e/tests/critical-path/browser/search-capabilities.e2e.ts (hunk 2 above)
Comment (AlenaNaboka) [reply tests/e2e/tests/critical-path/browser/search-capabilities.e2e.ts:177]: fixed

**Review by vlad-dargel — APPROVED** [review]

**Pull request merged**